code/parse/sexp/sexp_lookup.cpp

```diff
@@ -177,8 +179,19 @@ int add_dynamic_sexp(std::unique_ptr<DynamicSEXP>&& sexp, int type)
 	new_op.min = sexp->getMinimumArguments();
 	new_op.max = sexp->getMaximumArguments();
 
+	auto free_op_index = get_next_free_operator(sexp->getCategory());
```

Comment:
I'm not entirely comfortable with using an `auto` since its later used in a bitwise OR operation.  Should it resolve to a non-integer type for any reason it would invalidate the `new_op.value` later on.

LGTM otherwise.

Reply:
Good point. Changed.